src/lustre/lustreAstHelpers.ml

```diff
@@ -0,0 +1,599 @@
+(* This file is part of the Kind 2 model checker.
+
+   Copyright (c) 2020 by the Board of Trustees of the University of Iowa
```

Comment:
Since this new file only includes old code, you should write the year when the code was originally written (2015). If you add new code later, you should update the copyright line as follows:

>  Copyright (c) 2015, 2020 by [...]

Reply:
This is fixed now.